src/test/scala/com/gu/mobile/logback/JsonConcatenationLogicSpec.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.mobile.logback
+
+import org.specs2.matcher.Matchers
+import org.specs2.mutable.Specification
+
+class JsonConcatenationLogicSpec extends Specification with Matchers {
```

Comment:
Can the JSON be nested?

Reply:
Potentially and it should work, however I should add a test case for that shouldn't I?

I'll do that an release!